### DISCUSSION TITLE
Simple view page (for Family and Partners)

### DISCUSSION THREAD

**Discussion by mrahmadt** [firefly-iii/firefly-iii#8831] [Firefly III]
Hello

firefly-iii is very powerful for many people, but it can be overwhelming for some people

Here is my situation with my family. I have wife and 1 kid, and I created 2 bank accounts under my name.

Account A (House Expenses) - managed by my wife (She has the account card and buy everything using this card, I just transfer X amount every month to this account)
Account B (For my son, basically encourage him to save money and set different saving targets to buy (Games, trips...)

PS: I oversimplified the situation to focus on the idea, please don't suggest to open account to each of them :)

What do I need? firefly-iii is overwhelming for them, they just need a simplified page for them to login and get following:

Account A - The wife need a page to check how much did we spend so far (current balance), what did we buy (Transactions, Top Transactions, Expense accounts...etc) - only for this account.

Account B - The kid need to check current balance and specific Piggy bank assigned to him.

I know most probably this is not available out of the box in firefly-iii, but I wanted to ask before I get my hand dirty and develop something using firefly-iii API (or if such thing already in the roadmap)

**Comment by mrahmadt** [comment]
BTW, I'm aware about https://docs.firefly-iii.org/explanation/more-information/what-its-not/#share-your-administration

The idea is more about simple view for the family and to check if it's in the roadmap or I have to create something using firefly-iii  powerful API ❤️

**Reply by JC5** [reply]
I'm afraid you'll have to build your own overview. There are also people who did this in Grafana and in Home Assistant.
